spectrum/xpub.py

```diff
@@ -118,6 +120,42 @@ def populate_editors(self):
         picker = XpubPeoplePicker(self._driver)
         picker.choose_some(2)
 
+        for index in range(0, 3):
+            self._send_input_to(self.CSS_SUGGEST_REVIEWER_NAME_TEMPLATE.format(index=index), 'Reviewer %d' % index, 'reviewer %d name' % index)
+            self._send_input_to(self.CSS_SUGGEST_REVIEWER_EMAIL_TEMPLATE.format(index=index), 'reviewer%d@example.com' % index, 'reviewer %d email' % index)
+
+    def next(self):
+        XpubNextButton(self._driver).follow()
+        return XpubInitialSubmissionDisclosurePage(self._driver)
+
+
+class XpubInitialSubmissionDisclosurePage(PageObject):
+    CSS_SUBMITTER_SIGNATURE = 'input[name="submitterSignature"]'
+    CSS_DISCLOSURE_CONSENT = 'input[name="disclosureConsent"]'
+    CSS_SUBMIT = 'button[data-test-id="submit"]'
+    CSS_CONFIRM = 'button[data-test-id="accept"]'
+
+    def acknowledge(self):
+        self._send_input_to(self.CSS_SUBMITTER_SIGNATURE, 'Josiah Carberry', 'submitter signature')
+        self._checkbox(self.CSS_DISCLOSURE_CONSENT)
+
+    def submit(self):
+        self._driver.find_element_by_css_selector(self.CSS_SUBMIT).click()
+        LOGGER.info('Submitted')
+        self._driver.find_element_by_css_selector(self.CSS_CONFIRM).click()
+        LOGGER.info('Confirmed modal')
+        # it's ok, implicit wait is not enough because there may already be a h1 on the page
+        WebDriverWait(self._driver, 10).until(lambda driver: self._on_thank_you())
+        LOGGER.info('Reached thank you page')
+
+    def _checkbox(self, selector):
+        # need to click on the parent <label>
```

Comment:
replicates https://github.com/elifesciences/elife-xpub/blob/develop/test/pageObjects/disclosure.js#L9 but it's not great